README.md

```diff
@@ -102,3 +112,49 @@ mnc2nii PATH_TO/avg.XXX.mnc PATH_TO/template_XXX.nii
 ~~~
 
 </details>
+
+### Steps to register a subject to the T2w dog template. 
+
+We picked `sub-HarshmanDobby` for this.
+
+#### Data
+
+**[with internal data access]** Required data is at the `data.neuro.polymtl.ca:datasets/template_dog_virginiatech` repository under the `rb/update_HB` branch
+
+**[with no internal data access]** Please download the below data, with only sub-HarshmanDobby and it's derivatives (contains no sensitive information)
+[reg_to_temp_bids.zip](https://github.com/user-attachments/files/16071719/reg_to_temp_bids.zip)
+
+1. Protocol to segment the spinal cord of the sub-HarshmanDobby
+```
+Command used : sct_deepseg_sc -i sub-HarshmanDobby_T2w.nii.gz -c t2 -o sub-HarshmanDobby_T2w_label-SC_seg.nii.gz
+Manual correction was done by Rohan Banerjee on the output from sct_deepseg_sc
+```
+
+2. Protocol to get the disc labels
+```
+sct_label_utils -i sub-HarshmanDobby_T2w.nii.gz -create-viewer 1:24 -o sub-HarshmanDobby_T2w_labels-disc.nii.gz
+```
+
+The following steps were done for the registration to the template (change parameters accordingly):
+```
+sct_register_to_template -i sub-HarshmanDobby_T2w.nii.gz -s sub-HarshmanDobby_T2w_label-SC_seg.nii.gz -ldisc sub-HarshmanDobby_T2w_labels-disc.nii.gz -t <PATH_TO_TEMPLATE_PARENT_FOLDER> -c t2 -s-template-id 2 -param step=1,type=seg,algo=centermassrot:step=2,type=im,algo=syn,metric=CC,iter=3 -ofolder HB_to_temp -qc qc_HB_to_temp
+```
+
+#### Results:
+QC report
+[qc_HB_to_temp.zip](https://github.com/user-attachments/files/16071820/qc_HB_to_temp.zip)
+
+<details>
+<summary>anat2template overlayed on template</summary>
+<br>
+
+https://github.com/spinalcordtoolbox/template-dog/assets/25586344/f753a5b7-cf0b-4882-8137-1bf1072f3ea4
+</details>
+
+<details>
+<summary>template2anat overlayed on template</summary>
+<br>
+
+https://github.com/spinalcordtoolbox/template-dog/assets/25586344/11743718-fc78-4364-8d79-f4005b2f7399
```

Comment:
don't use video, it is cumbersome and it takes more space than required-- generate 2-image GIF instead